GitCommands/Git/GitModule.cs

```diff
@@ -3141,12 +3141,12 @@ public IReadOnlyList<string> GetAllBranchesWhichContainGivenCommit(ObjectId obje
                 return Array.Empty<string>();
             }
 
-            var result = exec.StandardOutput.Split(new[] { '\r', '\n', '*', '+' }, StringSplitOptions.RemoveEmptyEntries);
+            string[] result = exec.StandardOutput.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
It would be good to add a comment explaining what information git may provide and what transformation we do.
IIRC, `*` denotes the current branch, but I can't remember what `+` may be used for...

Reply:
@RussKie  that would be a link to Git documentation
Documentation is poor, does not statate that the line starts with two chars before branch
So something should be added below (this line is fine).

https://git-scm.com/docs/git-branch#_description

+ is used for linked work trees, similar to *